Keypad/Keypad.module.less

```diff
@@ -0,0 +1,95 @@
+// Keypad.module.less
+//
+@import "../styles/skin.less";
+@import "../styles/mixins.less";
+
+.keypad {
+	height: 510px;
+	width: 480px;
```

Comment:
These (and several more below) are pretty specific values. Could we move them into skin variables? This section will need to be wrapped in an `.applySkins` mixin too.

Reply:
I left `display`, `justify-content`, `position` as specific values. Would you like to move them too in skin variables?

Reply:
I think those rules are ok to keep as fixed.

Reply:
Any progress update on moving these `height` and `width` values to the skin section and conversion to variables?